fe/fe-core/src/main/java/com/starrocks/connector/iceberg/rest/IcebergRESTCatalog.java

```diff
@@ -194,6 +213,35 @@ public void renameTable(String dbName, String tblName, String newTblName) throws
         delegate.renameTable(TableIdentifier.of(dbName, tblName), TableIdentifier.of(dbName, newTblName));
     }
 
+    @Override
+    public boolean createView(ConnectorViewDefinition definition, boolean replace) {
+        Schema schema = IcebergApiConverter.toIcebergApiSchema(definition.getColumns());
+        ViewBuilder viewBuilder = delegate.buildView(TableIdentifier.of(definition.getDatabaseName(), definition.getViewName()));
+        viewBuilder = viewBuilder.withSchema(schema)
+                .withQuery("starrocks", definition.getInlineViewDef())
+                .withDefaultNamespace(Namespace.of(definition.getDatabaseName()))
+                .withDefaultCatalog(definition.getCatalogName())
+                .withProperties(buildProperties(definition))
+                .withLocation(defaultTableLocation(definition.getDatabaseName(), definition.getViewName()));
```

Comment:
do we support ceate view at specified location?

Reply:
to be honest, I am not sure why the specified location doesn't work. it will use an uuid as table location.